::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
: Copyright (c) 2018 Mirantis Inc., Enea AB and others.
:
: All rights reserved. This program and the accompanying materials
: are made available under the terms of the Apache License, Version 2.0
: which accompanies this distribution, and is available at
: http://www.apache.org/licenses/LICENSE-2.0
::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
From: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
Date: Sun, 25 Feb 2018 03:43:49 +0100
Subject: [PATCH] network.dpdk: Move ifcfg-br-prv to interfaces.u

Workaround for Upstream-Bug:
https://github.com/saltstack/salt/issues/40262

Signed-off-by: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
---
 linux/network/dpdk.sls | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/linux/network/dpdk.sls b/linux/network/dpdk.sls
index 05fe05f..420b038 100644
--- a/linux/network/dpdk.sls
+++ b/linux/network/dpdk.sls
@@ -128,7 +128,7 @@ linux_network_dpdk_bridge_interface_{{ interface_name }}:
         After=openvswitch-switch.service

 {# enforce ip address and mtu for ovs dpdk br-prv #}
-/etc/network/interfaces.d/ifcfg-{{ interface_name }}:
+/etc/network/interfaces.u/ifcfg-{{ interface_name }}:
   file.managed:
     - contents: |
         auto {{ interface_name }}
